web-services/client/src/main/java/datawave/webservice/query/QueryImpl.java

```diff
@@ -281,6 +284,11 @@ public boolean isMaxResultsOverridden() {
         return parameters == null ? null : Collections.unmodifiableSet(parameters);
     }
     
+    @Override
+    public long getQueryExecutionForCurrentPageStartTime() {
+        return this.queryExecutionForCurrentPageStartTime;
+    }
+    
     public void setQueryLogicName(String name) {
```

Comment:
You're right.  I've removed both the getter and setter from here, and `Query.java` and removed the unnecessary call to the setter in `DefaultQueryPlanner` since the currentPageStartTime  is tracked from RunningQuery -> logic -> transformer.  Thanks!